src/module/api/user/update/index.ts

```diff
@@ -34,9 +34,9 @@ export async function Update(updateUser: IUpdateUser): Promise<IUser[]> {
   }
 
   if (updateUser.title) {
-    nameObjJsnPatch.setOpe("replace");
-    nameObjJsnPatch.setPat("/obj/property/desc");
-    nameObjJsnPatch.setVal(updateUser.title);
+    titleObjJsnPatch.setOpe("replace");
```

Comment:
Had a surprisingly difficult time seeing that this was the wrong variable being used here.